src/crate/client/connection.py

```diff
@@ -122,6 +134,13 @@ def connect(servers=None,
     :param timeout:
         (optional)
         define the retry timeout for unreachable servers in seconds
+    :param backoff_factor:
+        (optional)
+        define the retry interval for unreachable servers in seconds
+    :param pool_size:
+        (optional)
+        Number of connections to save that can be reused.
+        More than 1 is useful in multithreaded situations.
```

Comment:
I would adjust the order of the parameter documentation to the order of the arguments (move at the end).

Reply:
Thanks. I've adjusted the parameter order within the documentation as suggested.